modules/virt-creating-vm-from-template.adoc

```diff
@@ -6,30 +6,34 @@
 [id="virt-creating-vm-from-template_{context}"]
 = Creating a VM from a template
 
-You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console.
+You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console. You can customize template or VM parameters, such as data sources, Cloud-init, or SSH keys, before you start the VM.
 
-Optional: You can customize template or VM parameters, such as data sources, cloud-init, or SSH keys, before you start the VM.
+You can choose between two views in the web console to create the VM:
+
+* A virtualization-focused view, which provides a concise list of virtualization-related options at the top of the view
+* A general view, which provides access to the various web console options, including Virtualization
 
 .Procedure
 
-. Navigate to *Virtualization* -> *Catalog* in the web console.
-. Click *Boot source available* to filter templates with boot sources.
+. From the {product-title} web console, choose your view:
+** For a virtualization-focused view, select *Administrator* -> *Virtualization* -> *Catalog*.
 +
-The catalog displays the default templates. Click *All Items* to view all available templates for your filters.
-
+** For a general view, navigate to *Virtualization* -> *Catalog*.
+. Click the *Template catalog* tab.
+. Click the *Boot source available* checkbox to filter templates with boot sources. The catalog displays the default templates.
+. Click *All templates* to view the available templates for your filters.
+. To focus on particular templates, enter the keyword in the `Filter by keyword` field.
+. Choose a template project from the *All projects* dropdown menu, or view all projects.
```

Comment:
I wonder if steps 5 and 6 should be sub-steps to 4. After you get all the templates, you can sort them using either method:

```suggestion
. Click *All templates* to view the available templates for your filters.
.. To focus on particular templates, enter the keyword in the `Filter by keyword` field.
.. Choose a template project from the *All projects* dropdown menu, or view all projects.
```